src/main/kotlin/app/revanced/patches/all/connectivity/fakewifi/patch/FakeWifiPatch.kt

```diff
@@ -0,0 +1,252 @@
+package app.revanced.patches.all.connectivity.fakewifi.patch
```

Comment:
This patch should be in the package `app.revanced.patches.all.connectivity.wifi.fake.patch`

Reply:
Or `app.revanced.patches.all.connectivity.wifi.spoof.patch` as suggested by FineFindus?

Reply:
Works.

Reply:
done